spec/12-the-scala-standard-library.md

```diff
@@ -642,6 +642,12 @@ object Predef {
   def classOf[T]: Class[T] = null
    // this is a dummy, classOf is handled by compiler.
 
+  // valueOf -----------------------------------------------------------
+
+  /** Retrieve the single value of a type with a unique inhabitant. */
+  @inline def valueOf[T](implicit vt: ValueOf[T]): T {} = vt.value
```

Comment:
> Now, as per @nafg's comment, classOf needs to be magic for an unrelated reason. But valueOf doesn't need to.

It still needs to be consistent. If `classOf` doesn't take a `ClassTag` it doesn't need, `valueOf` should not either.